### DISCUSSION TITLE
CPU only

### DISCUSSION THREAD

**Discussion by magimyster** [lllyasviel/stable-diffusion-webui-forge#295] [Q&A] [answered]
is it possible to use it only with cpu?

**Comment by huchenlei** [comment, marked as answer]
Yes. You can launch forge with `--always-cpu` cmd arg.

**Reply by magimyster** [reply]
wow thank you!

**Reply by Mythodim** [reply]
can you show me how to do it? i got the stable diffusion webui forge installed,

**Reply by Mythodim** [reply]
nvm https://zahiralam.com/blog/installing-and-running-stable-diffusion-webui-forge-on-cpu-only-ubuntu-systems/
